### ISSUE TITLE
Contributing guidelines not rendered

### ISSUE THREAD

**Issue by SirAionTech** [OCA/odoo-community.org#147]
## Module
No module

## Describe the bug
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst id shown as plain text, not rendered: 
![image](https://github.com/OCA/odoo-community.org/assets/143788994/784f4bba-0f72-4269-ad46-ff8aa682c223)

## To Reproduce
**Affected versions**: `master`

1. Go to https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst

**Expected behavior**
The rendered page should be shown.

**Comment by rousseldenis** [comment]
@SirAionTech Indeed. Checked the commit history and previous versions. The rendering is still broken.

Maybe Github has changed its rendering engine and does not support some formattings anymore (modules' READMEs are still ok)

**Comment by rousseldenis** [comment]
@OCA/core-maintainers @OCA/board

**Comment by yajo** [comment]
What if we convert it to markdown? It'll be easier to maintain for everybody and... render

**Comment by luisg123v** [comment]
> What if we convert it to markdown? It'll be easier to maintain for everybody and... render

I agree on that.

If it's OK, I could take care of converting it to markdown syntax.

**Comment by yajo** [comment]
Please

**Comment by StefanRijnhart** [comment]
Inline backticks are not rendered, it seems.

![image](https://github.com/OCA/odoo-community.org/assets/1033124/6980fe79-eda8-4be9-baa0-04a1d4085bfb)

**Comment by SirAionTech** [comment]
> Inline backticks are not rendered, it seems.
> 
> ![image](https://private-user-images.githubusercontent.com/1033124/318069511-6980fe79-eda8-4be9-baa0-04a1d4085bfb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE3MjMwNjQsIm5iZiI6MTcxMTcyMjc2NCwicGF0aCI6Ii8xMDMzMTI0LzMxODA2OTUxMS02OTgwZmU3OS1lZGE4LTRiZTktYmFhMC0wNGExZDQwODViZmIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMyOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMjlUMTQzMjQ0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MjgzMjYzNmE4MTkyNzFiYzQyZTg5MzQzNjUxMmNiYTc4ZTJiYTE5ZjI1ZWYwZjJhMDE1MTYyZTM3OTQ0ZGIxYiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.CcH9vBRUESvmQEM-XC3xJR-Fhul46BteW8tEZyqSkac)

ah didn't notice that, and also code blocks with underlined code is awful in my opinion..
![image](https://github.com/OCA/odoo-community.org/assets/143788994/f67e1178-678c-406a-9612-3f039bcd975d)

Not sure if that's enough to reopen this issue though; maybe another issue for these details?

**Comment by StefanRijnhart** [comment]
Well, apart from these issues, I still don't understand why the website page https://odoo-community.org/page/contributing redirects to the code on Github rather than rendering the page as a website page. I was hoping that is what would be fixed for this ticket as I think it looked better before. Do you know why that is not the case anymore?

**Comment by SirAionTech** [comment]
> Well, apart from these issues, I still don't understand why the website page https://odoo-community.org/page/contributing redirects to the code on Github rather than rendering the page as a website page. I was hoping that is what would be fixed for this ticket as I think it looked better before. Do you know why that is not the case anymore?

As far as I can remember, I've always only seen the guidelines as a `.rst` file in github, in https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst.

**Comment by StefanRijnhart** [comment]
Oh ok, maybe I misremember.

**Comment by yajo** [comment]
Yes, probably Github's built-in renderer broke at some point and they fixed it later.

**Comment by dreispt** [comment]
> Well, apart from these issues, I still don't understand why the website page https://odoo-community.org/page/contributing redirects to the code on Github rather than rendering the page as a website page. I was hoping that is what would be fixed for this ticket as I think it looked better before. Do you know why that is not the case anymore?

I don't think this was the case before, but it would be a welcome improvement.
It would probably modifying this repo to have it published as a website, and have odoo-community.org link to the published web page instead.

**Comment by pedrobaeza** [comment]
That was done a lot of years ago, and I think you Daniel, as well as @sbidoul, were involved to have canonical URLs for these pages. They were pointed through redirections then to the proper place.
